core/src/main/java/org/apache/accumulo/core/rpc/TTimeoutTransport.java

```diff
@@ -63,7 +64,8 @@ private TTimeoutTransport() {}
    * @throws IOException
    *           If the transport fails to be created/connected
    */
-  public static TTransport create(HostAndPort addr, long timeoutMillis) throws IOException {
+  public static TTransport create(HostAndPort addr, long timeoutMillis)
+      throws IOException, TTransportException {
```

Comment:
@EdColeman had the idea to consolidate the exceptions here. I am going to change it so we only have to throw one here and in similar places.